assets/src/edit-story/utils/useAddScrollbarWidth.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useLayoutEffect, useState } from 'react';
+
+function useAddScrollbarWidth(refContainer) {
```

Comment:
Why is this ref an argument and not something created in the hook and returned by it?

Reply:
Because it's used elsewhere as well and not only related to the scrollbar width directly. Seemed to be better-suited in the original place. Thoughts?

Reply:
Although maybe the `useAddScrollbarWidth` should be renamed instead.

Reply:
PTAL at 96cbce8